libs/coin-evm/src/signOperation.ts

```diff
@@ -67,21 +68,30 @@ export const buildSignOperation =
           erc20: true,
           domains: transaction.recipientDomain ? [transaction.recipientDomain] : [],
         };
+        const loadConfig: LoadConfig = {
+          cryptoassetsBaseURL: getEnv("DYNAMIC_CAL_BASE_URL"),
+          nftExplorerBaseURL: getEnv("NFT_ETH_METADATA_SERVICE") + "/v1/ethereum",
```

Comment:
I'm seeing a `if (isNFTActive(account.currency))` condition in the ethereum family, don't we need that here too?

Reply:
Nope, providing the URI or not won't change anything as long as the `nft: true` is not added to the `ResolutionConfig` (it will be added properly in the NFT PR right here: https://github.com/LedgerHQ/ledger-live/blob/e1d75bc992b707add781369aa1c93464f675c8c1/libs/coin-evm/src/signOperation.ts#L69)